sklearn/inspection/_plot/tests/test_boundary_decision_display.py

```diff
@@ -353,3 +339,54 @@ def test_dataframe_support(pyplot):
         # no warnings linked to feature names validation should be raised
         warnings.simplefilter("error", UserWarning)
         DecisionBoundaryDisplay.from_estimator(estimator, df, response_method="predict")
+
+
+@pytest.mark.parametrize("response_method", ["predict_proba", "decision_function"])
+def test_class_of_interest(pyplot, response_method):
```

Comment:
```suggestion
def test_class_of_interest_multiclass(pyplot, response_method):
```

and then please add a new test for binary classification problem with `set(y) == {"a", "b"}` with `class_of_interest="a"` or `"b"` for instance.